### ISSUE TITLE
txt2img2img doesn't working

### ISSUE THREAD

**Issue by nistvan86** [ThereforeGames/unprompted#55]
This was already mentioned here: https://github.com/ThereforeGames/unprompted/issues/53#issuecomment-1404646410

The example `[file common/examples/txt2img2img]` doesn't work. The following error is printed in the console and the second generated image is not an img2img image, just a normal txt2img of Walter White:

```
Error running process: D:\stable-diffusion\stable-diffusion-webui\extensions\unprompted\scripts\unprompted.py 1.98it/s]
Traceback (most recent call last):
  File "D:\stable-diffusion\stable-diffusion-webui\modules\scripts.py", line 386, in process
    script.process(p, *script_args)
TypeError: Scripts.process() missing 2 required positional arguments: 'is_enabled' and 'unprompted_seed'
```

---

Also with one of my own prompts I'm getting this error as well, I'm not sure if it's related, the demo template doesn't print this. But my script also doesn't work (it generates the source image and the mask, but img2img is not using the mask at all).

```Error running postprocess: D:\stable-diffusion\stable-diffusion-webui\extensions\unprompted\scripts\unprompted.py
Traceback (most recent call last):
  File "D:\stable-diffusion\stable-diffusion-webui\modules\scripts.py", line 404, in postprocess
    script.postprocess(p, processed, *script_args)
  File "D:\stable-diffusion\stable-diffusion-webui\extensions\unprompted\scripts\unprompted.py", line 431, in postprocess
    Unprompted.shortcode_objects[i].after(p,processed)
  File "D:\stable-diffusion\stable-diffusion-webui\extensions\unprompted/shortcodes\stable_diffusion\txt2mask.py", line 228, in after
    overlayed_init_img = draw_segmentation_masks(pil_to_tensor(p.init_images[0]), pil_to_tensor(self.image_mask.convert("L")) > 0)
AttributeError: 'StableDiffusionProcessingTxt2Img' object has no attribute 'init_images'
```

The prompt was (replaced actual prompts with <<placeholders>>):

    [set negative_prompt]<<negative prompt for a person>>[/set]
    [set cfg_scale]7[/set]
    [set sampling_steps]40[/set]
    [txt2img]<<prompt of a person>>[/txt2img]
    [after]
	    {sets prompt="<<img2img prompt of a person>>" denoising_strength=0.98 mask_blur=4 sampling_steps=140 cfg_scale=11}
        {txt2mask padding=40 neg_padding=10 smoothing=20 neg_smoothing=10 negative_mask="head" show}person{/txt2mask}
	    {img2img}
    [/after]

WebUI versions from footer:
python: 3.10.9  •  torch: 1.13.1+cu117  •  xformers: 0.0.16rc425  •  gradio: 3.16.2  •  commit: [91c8d0dc](https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/91c8d0dcfc9a5d46aec47fd3eb34c633c104b5bb)  •  checkpoint: [fe4efff1e1](https://google.com/search?q=fe4efff1e174c627256e44ec2991ba279b3816e364b49f9be2abc0b3ff3f8556)

**Comment by ThereforeGames** [comment]
Hi @nistvan86,

Thank you for reaching out!

The txt2img2img template has been fixed in Unprompted v7.1.0. Please give it a try and let me know if it resolves the issue with your own prompt as well.

Bonus: you can now use txt2img2img through the Wizard if you prefer a GUI.

![image](https://user-images.githubusercontent.com/95403634/217345391-23a72c83-1806-41b2-b924-dc57a3dbbf8c.png)

**Comment by nistvan86** [comment]
Tested it yesterday, but forgot to reply. It's working fine now, thank you :)
